test/ui-test/contentCreatorUiTest.ts

```diff
@@ -138,3 +138,71 @@ describe("Test Ansible sample execution environment file scaffolding", () => {
     );
   });
 });
+
+describe("Test collection plugins scaffolding", () => {
+  let createButton: WebElement;
+  let editorView: EditorView;
+
+  async function testWebViewElements(
+    command: string,
+    editorTitle: string,
+    pluginName: string,
+    pluginType: string,
+  ) {
+    await workbenchExecuteCommand(command);
+    await sleep(10000);
+
+    await new EditorView().openEditor(editorTitle);
+    const webview = await getWebviewByLocator(
+      By.xpath("//vscode-text-field[@id='plugin-name']"),
+    );
+
+    const pluginNameTextField = await webview.findWebElement(
+      By.xpath("//vscode-text-field[@id='plugin-name']"),
+    );
+    expect(pluginNameTextField, "pluginNameTextField should not be undefined")
+      .not.to.be.undefined;
+    await pluginNameTextField.sendKeys(pluginName);
+
+    const pluginTypeDropdown = await webview.findWebElement(
+      By.xpath("//vscode-dropdown[@id='plugin-dropdown']"),
+    );
+    expect(pluginTypeDropdown, "pluginTypeDropdown should not be undefined").not
+      .to.be.undefined;
+    await pluginTypeDropdown.sendKeys(pluginType);
+
+    const overwriteCheckbox = await webview.findWebElement(
+      By.xpath("//vscode-checkbox[@id='overwrite-checkbox']"),
+    );
+    expect(overwriteCheckbox, "overwriteCheckbox should not be undefined").not
+      .to.be.undefined;
+    await overwriteCheckbox.click();
+
+    createButton = await webview.findWebElement(
+      By.xpath("//vscode-button[@id='create-button']"),
+    );
+    expect(createButton, "createButton should not be undefined").not.to.be
+      .undefined;
+
+    expect(
+      await createButton.isEnabled(),
```

Comment:
@shatakshiiii Can you add a test section to unselect overwrite and rescaffold (expected failure) for slightly more test coverage? See this change for reference: https://github.com/ansible/vscode-ansible/pull/1733/files#diff-87cfb0aafd5395ec414531d5cf42578e92cb4995438776ed01207c68c95c6ad0R152-R156